rules/integrations/aws/resource_development_sns_topic_created_by_rare_user.toml

```diff
@@ -0,0 +1,149 @@
+[metadata]
+creation_date = "2025/02/11"
+integration = ["aws"]
+maturity = "production"
+updated_date = "2025/02/11"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies when an SNS topic is created by a user who does not typically perform this action. Adversaries may create SNS
+topics to stage capabilities for data exfiltration or other malicious activities. This is a [New
+Terms](https://www.elastic.co/guide/en/security/current/rules-ui-create.html#create-new-terms-rule) rule that only flags
```

Comment:
Markdown is not rendered in the description afaik

Reply:
oh, I did not realize that, thanks Jon!

Reply:
fixed; added the link and note to investigation guide